hi-ml-multimodal/src/health_multimodal/image/model/pretrained.py

```diff
@@ -84,3 +86,39 @@ def get_biovil_t_image_encoder() -> ImageModel:
         pretrained_model_path=biovilt_checkpoint_path,
     )
     return image_model
+
+
+def get_imagenet_init_encoder() -> ImageModel:
```

Comment:
Would you mind adding a simple unit-test checking if this function returns the expected image encoder?